### DISCUSSION TITLE
Osu!lazer leaderboard scores not updating

### DISCUSSION THREAD

**Discussion by Chae-Seul** [ppy/osu#22863] [Q&A] [answered]
Hello, it's my first time making a forum post, I'm just really lost in the moment. I have been playing for so many hours just to notice that my scores aren't updating in the leaderboard nor personal scores.

I have been playing osu!lazer for some time... I do appear in the leaderboard sometimes, so I have a huge problem right now: My scores are not updating, nor appear in the leaderboard. It's in my recent games in my profile, it even shows the pp, but it does not update in the osu!lazer global leaderboard... I DID NOT PLAY ON MULTIPLAYER.

https://imgur.com/a/uZ4C6Hw

![osuproblem](https://user-images.githubusercontent.com/81992281/224879018-b63f6632-e37c-42bc-8621-6d41c386f545.png)

as you can see, my last score indicated was 17 days, but I made an S score in solo mode an hour ago. So yeah. Pretty saddening. What may be the problem? I tried reinstalling, and it didn't work at all.

Oh by the way, I'm using version 2023.305.0 lazer

**Comment by peppy** [comment, marked as answer]
Could you please check again now?

**Reply by Chae-Seul** [reply]
Peppy, thank you very much.
I was playing for hours and dying in sadness after knowing nothing was saving.
I saw the updated scores now.
Thank you so much for everything.

**Reply by peppy** [reply]
Great to hear.

Keep in mind we are going to be resetting lazer scores in the near future, so don't get too attached!
